### PR TITLE
Add github ribbon

### PR DESCRIPTION
**Pull request by jimmyhchan**
because ribbons are cool. see https://github.com/blog/273-github-ribbons

### CODE REVIEW

**Comment by sethkinast** [comment]
Bad mobile experience if you use the imgribbon. Maybe http://codepo8.github.io/css-fork-on-github-ribbon/ ?

**Comment by smfoote** [comment]
We could just hide the ribbon completely on mobile.

**Comment by sethkinast** [comment]
I'm OK with that too.

**Comment by jimmyhchan** [comment]
Perhaps the bigger thing is that we should link to github

**Comment by smfoote** [comment]
We do link to GitHub in the "Contributing" link. But we can merge this, too.

**Comment by jimmyhchan** [comment]
derp. my screen was too short to see it.   updated to a version inspired by the codepo8 CSS one.  plus updated the readme to get it working with bundler

File: README.md [hunk 1]
```diff
@@ -9,7 +13,7 @@ We use SCSS/Compass/archetype.
 2. Install [Compass](http://compass-style.org/install/)
 3. Install [archetype](http://www.archetypecss.com/documentation/getting-started/)
 
-After making chages to a file in the scss directory, run `compass compile` from the project's root directory. If you are planning on making lots of changes to scss files, you can run `compass watch` and the CSS compilation will happen as you save the scss files.
+After making chages to a file in the scss directory, run `bundle exec compass comile` or `compass compile` from the project's root directory. If you are planning on making lots of changes to scss files, you can run `compass watch` and the CSS compilation will happen as you save the scss files.
```
Comment (sethkinast) [inline README.md]: comile -> compile

**Comment by sethkinast** [comment]
:+1:

**Comment by sethkinast** [comment]
Looks good, but could we use absolute position instead of fixed? it can get in the way of the examples.

**Pull request merged**